spring-integration-file/src/main/java/org/springframework/integration/file/remote/RemoteFileOperations.java

```diff
@@ -155,6 +155,6 @@
 	 * @return a session.
 	 * @since 4.3
 	 */
-	Session<F> getSesssion();
+	Session<F> getSession();
```

Comment:
He-he! That was introduced exactly recently in the RC: https://jira.spring.io/browse/INT-4015
So, I even don't know, since currently we are with GA...